profiles/thurloe-configs/terraform/thurloe/config.tf

```diff
@@ -0,0 +1,42 @@
+provider "vault" {}
+
+resource "vault_token" "render_token" {
+  policies = ["${var.config_render_vault_policy}"]
+
+  renewable = true
+  ttl = "20m"
+}
+
+resource "null_resource" "config" {
+  triggers = {
+    always = "${uuid()}"
+  }
+  provisioner "local-exec" {
+    command = "ruby /workbench/render_configs.rb"
+    environment = {
+      INSTANCES = "${jsonencode(data.google_compute_instance_group.service-instances.instances)}"
+      CONFIG_BUCKET = "${var.config_bucket_name}"
+      VAULT_TOKEN = "${vault_token.render_token.client_token}"
+      GITHUB_TOKEN_VAULT_PATH = "${var.github_token_vault_path}"
+      INSTANCE_TYPE = "${var.instance_type}"
+      RUN_CONTEXT = "${var.run_context}"
+      APP_NAME = "${var.app_name}"
+      ENV = "${var.environment}"
+      GIT_REPO = "${var.git_repo}"
+      INSTANCE_NAME = "thurloe01"
```

Comment:
remove  the hardcoding here, allow this to be a  var too?

Reply:
thanks, this is a leftover and can be removed, using `INSTANCES` to capture all the instances.